llvm/test/CodeGen/X86/windows-seh-EHa-PreserveCFG.ll

```diff
@@ -0,0 +1,80 @@
+; RUN: llc -mtriple=x86_64-pc-windows-msvc %s
```

Comment:
This RUN line seems to leave a .s file around that causes failures in subsequent runs:
https://lab.llvm.org/buildbot/#/builders/139/builds/56922
```
******************** TEST 'LLVM :: CodeGen/X86/windows-seh-EHa-PreserveCFG.s' FAILED ********************
Test has no 'RUN:' line
********************
```
Is this actually testing anything? Or is it just trying to verify the compiler does not crash?

Reply:
@HaohaiWen Could you restructure this test so that it doesn't generate the .s file (maybe redirect the output to /dev/null or something similar?) This is causing unresolved test errors on several of the buildbots I manage.

Reply:
Oh, sorry. I fixed it in https://github.com/llvm/llvm-project/pull/77784.
It's verifying compiler not crash.